File: tooling/sdkgen/test/typescript/openws-sdkgen-typescript-node/chat/core/src/models/client/received-message-payload.ts
export interface ReceivedMessagePayloadInit {
    senderId: string
    roomId: string
    text: string
}

export class ReceivedMessagePayload implements ReceivedMessagePayloadInit {
    readonly senderId: string
    readonly roomId: string
    readonly text: string

    constructor(
        { senderId, roomId, text }: ReceivedMessagePayloadInit = {} as ReceivedMessagePayloadInit
    ) {
        this.senderId = senderId
        this.roomId = roomId
        this.text = text
    }
}

File: tooling/sdkgen/test/dotnet/UnityTestbed/Assets/Example.Chat.Sdk/Runtime/Example.Chat.Core.Sdk/Models/Client/ReceivedMessagePayload.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace Example.Chat.Core.Models.Client
{
    public partial class ReceivedMessagePayload
    {
        [JsonProperty("senderId")]
        public string SenderId;

        [JsonProperty("roomId")]
        public string RoomId;

        [JsonProperty("text")]
        public string Text;

    }
}